### PR TITLE
Flatten

### PR DESCRIPTION
**Pull request by fiunchinho**
Towards https://github.com/giantswarm/giantswarm/issues/7819

This PR will merge all the work in progress to achieve a flattened operator.

### CODE REVIEW

File: service/controller/resource/instance/create.go [hunk 1]
```diff
@@ -7,14 +7,14 @@ import (
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/operatorkit/controller/context/reconciliationcanceledcontext"
 
-	"github.com/giantswarm/azure-operator/service/controller/v14/key"
-	"github.com/giantswarm/azure-operator/service/controller/v14/resource/instance/internal/state"
+	"github.com/giantswarm/azure-operator/service/controller/key"
+	state2 "github.com/giantswarm/azure-operator/service/controller/resource/instance/internal/state"
```
[the remark this answers is not in the document]
Comment (fiunchinho) [reply service/controller/resource/instance/create.go]: Yeah, probably the `IDE`. Fixing it here https://github.com/giantswarm/azure-operator/pull/678

**Comment by whites11** [comment]
I guess the review has been done on the previous branch, right?

**Comment by fiunchinho** [comment]
Yeah, all the other PR's have been reviewed. I'm going to run the e2e tests just in case, and if they pass I'll merge it.

**Comment by whites11** [comment]
> Yeah, all the other PR's have been reviewed. I'm going to run the e2e tests just in case, and if they pass I'll merge it.

OK, approved to share the blame for when everything will explode
https://media.giphy.com/media/eVy46EWyclTIA/giphy.gif

**Pull request merged**